apps/fluent-tester/src/TestComponents/Shadow/ShadowButtonTestSection.tsx

```diff
@@ -11,7 +12,7 @@ const CustomFABShadow64 = FAB.customize({
 export const ShadowButtonTestSection: React.FunctionComponent = () => {
   if (Platform.OS === 'ios') {
     return (
-      <View style={stackStyle}>
+      <View style={{ backgroundColor: shadowTestPageBackgroundColor() }}>
```

Comment:
Nit: Inline style would force unnecessary re-renders

Reply:
Oh missed that, thanks for catching it

Reply:
Should be fixed now after changing to using themedStyleSheet